lib/esp/resources/tag.rb

```diff
@@ -32,12 +36,13 @@ def self.for_alert(alert_id = nil)
 
     # Find a Tag by id
     #
-    # ==== Parameter
-    #
-    # +id+ | Required | The ID of the tag to retrieve
+    # *call-seq* -> +super.find(id)+
     #
-    # :call-seq:
-    #  find(id)
+    # @overload find(id)
+    #   @param id [Integer, Numeric] Required ID of the tag to retrieve.
+    # @overload find(non_numeric, options = {})
```

Comment:
Not sure about documenting this overload, but if we do, it should be @overload find(options = {})